pkg/config/env.go

```diff
@@ -79,6 +79,34 @@ var Config = struct {
 	// RecorderType - the pipeline to log data records, e.g. Kafka
 	RecorderType string `env:"FLAGR_RECORDER_TYPE" envDefault:"kafka"`
 
+	// RecorderFrameOutputMode - indicates which data record frame output mode should we use.
+	// Possible values: payload_string, payload_raw_json
+	//
+	// payload_string mode - it respects the encryption settings, and it will stringify the payload to unify
+	//                the type of the output for both plaintext and encrypted payload.
+	//   {
+	//     "payload": "{\"evalContext\":{\"entityID\":\"123\"},\"flagID\":1,\"flagKey\":null,\"flagSnapshotID\":1,\"segmentID\":1,\"timestamp\":null,\"variantAttachment\":null,\"variantID\":1,\"variantKey\":\"control\"}",
+	//     "encrypted": false
+	//   }
+	//
+	// payload_raw_json mode - ignores the encryption settings.
```

Comment:
Since we have removed the `MarshalIdent` I think we should either put the example JSON in a single line (as in the string version) or make a note in the comment that the resulting output will consist of a single record per line.

Reply:
good catch, I changed to show the single-line version in the comments.